src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,90 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
 
-// import { MovieCard } from '../MovieCard';
+type Props = {
+  addNewMovie: (movie: Movie) => void;
+};
 
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
+export const FindMovie: React.FC<Props> = ({ addNewMovie }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState({} as Movie);
```

Comment:
```suggestion
  const [movie, setMovie] = useState<Movie | null>(null);
```
Используй null в качестве дефолтного значения